### PR TITLE
move custom action to new project

### PR DESCRIPTION
**Pull request by jimmidier**

### CODE REVIEW

File: src/SSW.SophieBot/SSWSophieBot.UtcConvertAction/SSWSophieBot.UtcConvertAction.csproj [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (PatrickZhao1989) [inline src/SSW.SophieBot/SSWSophieBot.UtcConvertAction/SSWSophieBot.UtcConvertAction.csproj:1]: Does this need to be a class lib? try net standard 2.0 or is this setup defined by Microsoft?

**Review by PatrickZhao1989 — CHANGES_REQUESTED** [review]

File: src/SSW.SophieBot/SSWSophieBot.UtcConvertAction/SSWSophieBot.UtcConvertAction.csproj (hunk 1 above)
Comment (jimmidier) [reply src/SSW.SophieBot/SSWSophieBot.UtcConvertAction/SSWSophieBot.UtcConvertAction.csproj:1]: Changed to net standard 2.1

**Review by PatrickZhao1989 — APPROVED** [review]
Good work

**Comment by PatrickZhao1989** [comment]
[Custom dialog action - external resource](https://trello.com/c/2Sl9MQnn/5-custom-dialog-action-external-resource)

**Comment by PatrickZhao1989** [comment]
[Custom dialog action - external resource](https://trello.com/c/2Sl9MQnn/5-custom-dialog-action-external-resource)

**Pull request merged**